gensim/models/wrappers/fasttext.py

```diff
@@ -220,6 +220,10 @@ def save(self, *args, **kwargs):
         super(FastText, self).save(*args, **kwargs)
 
     @classmethod
+    def load_word2vec_format(cls, *args, **kwargs):
+        return FastTextKeyedVectors.load_word2vec_format(*args, **kwargs)
```

Comment:
That is an odd error! I suspect it's not really the presence/absence of that method that triggered it, but something else either random or hidden in the whitespace.